frontend/src/lib/apiClient.js

```diff
@@ -14,6 +17,32 @@ const getRecipes = (search) => {
     .then(results => results.data);
 };
 
+const createUser = data => apiClient.post('api/v1/user', {
+  user: {
+    first_name: data.firstName,
+    last_name: data.lastName,
+    email: data.email,
+    password: data.password,
+  },
+});
+
+const loginUser = (data) => {
+  const credentials = { email: data.email, password: data.password };
+  return apiClient.post('api/v1/auth/login', {
+    user: credentials,
+  });
+};
+
+const getUser = userId => apiClient.get(`api/v1/user/${userId}`, {
+  headers: {
+    Authorization: `Bearer ${sessionStorage.jwt}`,
```

Comment:
I think the JWT was already coming back from the API with the `Bearer` string?

Reply:
You're absolutely right. fixed.